samples/CMakeLists.txt

```diff
@@ -84,6 +84,22 @@ if (CONFIG_NCS_INCLUDE_RPMSG_CHILD_IMAGE OR CONFIG_NCS_SAMPLE_EMPTY_NET_CORE_CHI
       "couldn't resolve which RPMsg sample to add.")
   endif()
 
+  if (CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU)
```

Comment:
`if (CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP)` ?
To automatically align the buffers between app and net cores

Reply:
yes, additionally I will use the imply syntax for the ```NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP``` option so that the user has the possibility to opt-out from this feature

Reply:
Great